### ISSUE TITLE
Empty collection does not let the plot become invisible

### ISSUE THREAD

**Issue by Daimonion1980** [beto-rodriguez/LiveCharts2#1505]
**Describe the bug**
I have a WPF UserControl which implements a Livecharts BasicLine Plot and is enabled to show one or more graphs. For this I implemented a ObservableCollection<ISeries> and two Axis Lists.  (I work with [MVVMGen](https://github.com/thomasclaudiushuber/mvvmgen) as boilerplate code generator)

```csharp
[Property] ObservableCollection<ISeries> plotData = [];
[Property] IList<ICartesianAxis> xAxis = [];
[Property] IList<ICartesianAxis> yAxis = [];
```

Adding entries to this setup works pretty well. The data appears in the UI. 
When removing an element (removing the specific entry in PlotData and YAxis) the data disappear in the ui.
But when removing the last entry and clearing all Collections and Lists, so that there Length is zero  
the UI held the last element and does not become invisible or collapsed. The label from the entry disappears but coordinates and the last graph is still available. 

When adding new Plots to the Collections and Axis Lists the UI is rebuild.

**Expected behavior**
When the collection is empty, the view shall be invisible.

**Screenshots**
after adding some data to the collections and lists:
![threee Graphs added](https://github.com/beto-rodriguez/LiveCharts2/assets/12880413/5573ee74-2fff-475a-b2d5-8bedaca8f0aa)

After removing (e.g. Plotdata.Clear(), XAxis.Clear(), YAxis.Clear()) all elements from the collections and lists:
![Collection empty](https://github.com/beto-rodriguez/LiveCharts2/assets/12880413/a785055f-2015-4be5-912b-1772fa0f5a5e)

**Comment by network-interface-controller** [comment]
Might be related to #1365

**Comment by Daimonion1980** [comment]
Yep, didn't found this because of bad search skills... ;)

**Comment by beto-rodriguez** [comment]
Hi @Daimonion1980 

I am not able to reproduce this issue, it seems that everything work as expected for me:

![clear](https://github.com/user-attachments/assets/0e0265ae-0481-4cd4-9eb2-0f87bc8b5235)

Can you please provide a complete and minimal example where I can reproduce the issue?

@network-interface-controller I think that is a different issue, I'l;l reply there with the details.

**Comment by beto-rodriguez** [comment]
I will close this for now because Iw as not able to reproduce, but feel free to reply or create a new issue if it persists.

**Comment by Daimonion1980** [comment]
Thanks for your response. At the moment I am building an app where I am able to show you this behavior. The original App is to big to show. 
I hope to be done this week and will open a new issue or response on this issue when I am not able to reproduce it via this app.

**Comment by Daimonion1980** [comment]
Okay, so I was able to reproduce it in a small app which you can download here: https://github.com/Daimonion1980/livechartsDemo/tree/main

At first I was not able to reproduce the behavior because I just added PlotData and did not mange the axes manually. But after adding the code to manage the axes and the ScalesYAt Property the last Plot is not removed correctly.
I mentioned this in https://github.com/beto-rodriguez/LiveCharts2/issues/1504

Please check the readme in the repo with the description to see the problem.

If you are not able to reopen this issue I can create a new one.

**Comment by beto-rodriguez** [comment]
Thanks for the repo!

I tried it and yes, I was able to reproduce the issue, this is related with a bug on toggling the series visibility, this is already fixed with #1656 and will be included in the next version of the library.

I will close this for now because this is fixed already in the dev branch and there is a similar issue #1164 where we can track this issue, thanks again!
